src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -193,10 +253,14 @@ def _download_file(
 
 def get_files_selection(dl_dir: Path) -> Dict[str, str]:
     """
-    Among all the files downloaded, only keep a subset for which we use a controlled name.
-    Return a dict[name] = file_path
-    The file_path is relative to the download dir
-    Current names are defined in _FILE_ENDS
+    Among all downloaded files only keep a subset for which we use a controlled name.
+    Current names are defined in dict: '_FILE_ENDS'
+
+    Args:
+        dl_dir: Local path to downloaded FTP files. The file_path is relative to the download dir.
+
+    Returns:
+        Dict: Subset target files. File name end (key)<=>file name(value)
```

Comment:
```suggestion
    """Returns a dictionary with the relevant downloaded files classified.

    Args:
        dl_dir: Local path to downloaded FTP files.

    Returns:
        Dictionary of file type (e.g.`"report"`) as keys and the relative file path (from `dl_dir`) as values.

    Raises:
        FileDownloadError: If `dl_dir` tree does not include a file named `*_assembly_report.txt`.
```
However, `FileDownloadError` should not be raised in this function, but in `get_root_name()` (based on the description and functionality of both methods).

Reply:
Note: watch out that I have included the `"""` in my suggestion when they are in a separate line in the original code.